components/Slots/Slots.jsx

```diff
@@ -0,0 +1,55 @@
+import styles from "./Slots.module.css";
+import en from "javascript-time-ago/locale/en";
+import TimeAgo from "javascript-time-ago";
+
+TimeAgo.addDefaultLocale(en);
+
+const timeAgo = new TimeAgo("en-US");
+const DEFAULT_BRANCH_NAME = "main";
+
+export function Slots({ slots = [], url }) {
+  const slotsWithoutBot = slots.filter((s) => s.commitAuthor !== "web-flow");
+
+  return (
+    <ul className={styles.slots}>
+      {slotsWithoutBot.map(
+        ({
+          branch,
+          slotUrl,
+          date,
+          commitMessage,
+          commitUrl,
+          commitAuthor,
+          commitAuthorAvatar,
+        }) => (
+          <li className={styles.slot} key={commitUrl}>
+            <a
+              href={branch === DEFAULT_BRANCH_NAME ? url : slotUrl}
+              target="_blank"
+              rel="noreferrer"
+              className={styles.slot__link}
+            >
+              {branch}, {timeAgo.format(new Date(date))}
```

Comment:
Let's separate branch and time. 

By the way, time will be very good to put in `<time datetime="ISO-raw-date-format"></time>`

Reply:
![image](https://user-images.githubusercontent.com/22644149/211168724-38d982e4-2af7-4b8f-ad14-9c83603b38ed.png)

\+ And we will have fewer underline, which we seem to have overdone

Reply:
move to time at in the https://github.com/ekaterinburgdev/projects/pull/10